### PR TITLE
Improve documentation formatting

### PR DESCRIPTION
**Pull request by KyleWJohnston**
This mostly fixes documentation formatting, but there are a few minor changes to the new Docker commands and templates.

### CODE REVIEW

File: docs/install/docker.md [hunk 1]
```diff
@@ -60,6 +61,8 @@ services:
   web_recipes:
     image: vabene1111/recipes
     restart: always
+    ports:
+      - 80:8080
```
Comment (vabene1111) [inline docs/install/docker.md:65]: this is not correct. If the web_recipes container already exposts port 80 nginx can no longer do so

**Review by vabene1111 — CHANGES_REQUESTED** [review]
i have added a small comment, if you could quickly remove these two lines i can merge this and the documentation will be in good shape again

**Comment by vabene1111** [comment]
I will also review the other branches in the evening to make sure they dont suffer from similar issues.

File: docs/install/docker.md (hunk 1 above)
Comment (KyleWJohnston) [reply docs/install/docker.md]: This makes sense, thank you. I did some further testing and think I've figured it (Docker basics) out.

For example, to expose on port 3000, you could use `docker run -d -p 3000:8080 ...` or, for Docker Compose,
```
  nginx_recipes:
    ports:
    - 3000:80
```

It worked in my tests.

**Review by vabene1111 — APPROVED** [review]

**Comment by vabene1111** [comment]
Thank you 👍 i will have a look at your other mentioned issue in the PR asap

**Pull request merged**